packages/vm/src/calls.rs

```diff
@@ -227,7 +227,7 @@ where
     let env = to_vec(env)?;
     let msg = to_vec(msg)?;
     let data = call_ibc_channel_open_raw(instance, &env, &msg)?;
-    let result: ContractResult<()> =
+    let result: ContractResult<Option<IbcV3ChannelOpenResponse>> =
```

Comment:
Ah nice, here we unify the two versions now into one type.

Reply:
Yup, the VM uses the backwards compatible ibc3 versions. Well noted we don't need a flag for this repo